storage/remote/queue_manager.go

```diff
@@ -170,18 +195,18 @@ func newQueueManagerMetrics(r prometheus.Registerer, rn, e string) *queueManager
 		Help:        "The number of shards that the queues shard calculation wants to run based on the rate of samples in vs. samples out.",
 		ConstLabels: constLabels,
 	})
-	m.bytesSent = prometheus.NewCounter(prometheus.CounterOpts{
+	m.samplesBytesTotal = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace:   namespace,
 		Subsystem:   subsystem,
-		Name:        "sent_bytes_total",
-		Help:        "The total number of bytes sent by the queue.",
+		Name:        "samples_bytes_total",
+		Help:        "The total number of bytes sent by the queue after compression.",
 		ConstLabels: constLabels,
 	})
-	m.maxSamplesPerSend = prometheus.NewGauge(prometheus.GaugeOpts{
+	m.metadataBytesTotal = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace:   namespace,
 		Subsystem:   subsystem,
-		Name:        "max_samples_per_send",
-		Help:        "The maximum number of samples to be sent, in a single request, to the remote storage.",
+		Name:        "metadata_bytes_total",
```

Comment:
I think we'd be better off with label here rather than metric names. Think about the query you'd need to calculate the total bandwidth usage. Exemplars may also end up as another set (or could end up with samples).

Reply:
That could happen alright.